trackingC.Rmd

```diff
@@ -567,6 +567,28 @@ ggplot(plot_data, aes(x = year, fill = def)) +
 
 ```
 
+``` {r af-trend, eval=FALSE}
+trend <- plot_data %>%
+  filter(year %in% 1960:2020)
+
+af_trend <-  trend %>%
+  group_by(ssp, def) %>%
+  do(mod = lm(med ~ year, data = .)) %>%
+  summarize(ssp = ssp,
+            def = def,
+            broom::tidy(mod)) %>%
+  filter(term == "year") %>%
+  mutate(decade = estimate * 10,
+         error = std.error * 10)
+
+clean_trend <- af_trend %>%
+  arrange(def) %>%
+  select(ssp, def, decade, error, p.value) %>%
+  rename("computation" = def, 
+         "trend" = decade)
```

Comment:
Perhaps end by doing `%>% knitr::kable()` to make a nice looking table in the output.